### ISSUE TITLE
V3.0 - still seeing errors on preview and build - sometimes

### ISSUE THREAD

**Issue by joelvarty** [graysonhicks/gatsby-plugin-loadable-components-ssr#24]
Here's the from CMS Preview on Gatsby Cloud.  
Hopefully, this is enough to help diagnose what's happening.

```
14:58:22 PM:
ERROR "gatsby-plugin-loadable-components-ssr" threw an error while running the onPostBuild lifecycle:
14:58:22 PM:
ENOENT: no such file or directory, unlink '/usr/src/app/www/public/loadable-stats-build-javascript.json'
14:58:22 PM:
  42 | const onPostBuild = () => {
14:58:22 PM:
  45 | };
14:58:22 PM:
> 44 |   (0, _fs.unlinkSync)(statsPath);
14:58:22 PM:
  46 |
14:58:22 PM:
  47 | exports.onPostBuild = onPostBuild;
14:58:22 PM:
  43 |   // Clean after ourselves
14:58:22 PM:
     |                      ^
14:58:23 PM:
14:58:23 PM:
  
14:58:23 PM:
  Error: ENOENT: no such file or directory, unlink '/usr/src/app/www/public/load  able-stats-build-javascript.json'
14:58:23 PM:
  
14:58:23 PM:
  - gatsby-node.js:44 Object.onPostBuild
14:58:23 PM:
    [www]/[gatsby-plugin-loadable-components-ssr]/gatsby-node.js:44:22
14:58:23 PM:
    [www]/[gatsby]/src/utils/api-runner-node.js:590:13
14:58:23 PM:
  
14:58:23 PM:
  - api-runner-node.js:434 runAPI
  ```

**Comment by joelvarty** [comment]
Wierd that this is erroring every 3rd build:

![image](https://user-images.githubusercontent.com/4520625/120700459-adc21300-c47f-11eb-9305-2f1250931a36.png)

**Comment by graysonhicks** [comment]
This `unlink` error should be resolved by #27 which just released under `3.1.0`.  Thanks!
